airbyte-integrations/connectors/source-zendesk-chat/source_zendesk_chat/spec.json

```diff
@@ -18,7 +18,7 @@
       "subdomain": {
         "type": "string",
         "title": "Subdomain",
-        "description": "Required if you access Zendesk Chat from a Zendesk Support subdomain.",
+        "description": "Required config which would be appended to the base url",
```

Comment:
@pnilan do you know if we can add a format requirement here to make sure people do NOT include `https://` in the beginning of the subdomain?

Reply:
We can specify an expected regex `pattern` for a given field. [Here's an example](https://github.com/airbytehq/airbyte/blob/89f2db4793123d6a11ab73ace2b2668f8f3b0fd0/airbyte-integrations/connectors/source-chargebee/source_chargebee/spec.yaml#L30). Maybe this would work?

```
"pattern": "^(?!https://)"
```

We can also provide an array of "examples" (see the next line of above link) which will show up in the field's tooltip in the builder.